sycl/test-e2e/KernelCompiler/opencl_queries.cpp

```diff
@@ -9,8 +9,8 @@
 // REQUIRES: ocloc && (opencl || level_zero)
 // UNSUPPORTED: accelerator
 
-// Fails with opencl and level_zero on linux, enable when fixed.
-// XFAIL: opencl || (linux && level_zero)
+// Fails with level_zero on linux, enable when fixed.
+// XFAIL: (linux && level_zero)
```

Comment:
Pretty sure this can be removed. It was added when the driver change triggered new failures ( https://github.com/intel/llvm/pull/13547 )

Your new WA should mean that the test should pass everywhere.

Reply:
@cperkinsintel do we want to proceed with this WA update? it will hide the problem with opencl reporting, not solve it

Reply:
removed XFAIL here https://github.com/intel/llvm/pull/13584/commits/2833f9ffc943ca598f26e6143f91b731b0f2d5a8